package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "angular-project",
```

Comment:
change project name to vanya innovations

Reply:
changed